docroot/themes/custom/mass_theme/mass_theme.theme

```diff
@@ -5982,6 +5982,19 @@ function mass_theme_preprocess_views_view_collection_all_page_all(&$variables) {
   $tid = reset($res);
   $term = Term::load($tid);
 
+  // Attach mg_labels metatag, if it's available.
```

Comment:
I'm a little surprised to see token replacement in mass_theme_preprocess_views_view_collection_all_page_all. Doesn't metatag have a more standard way that uses config?

Reply:
Removed this part and enabled views display extender of the metatag_views module. Configured it that way.